lib/bundle/brew_services.rb

```diff
@@ -0,0 +1,26 @@
+module Bundle
+  class BrewServices
+    def self.restart(name)
+      new(name).restart
+    end
+
+    def initialize(name)
+      ensure_brew_services_installed
+      @name = name
+    end
```

Comment:
Sorry, the first method in the class (e.g. before `restart`); just a typical convention.